lib/regaliator/response.rb

```diff
@@ -22,7 +22,7 @@ def data
     end
 
     def success?
-      @success ||= http_response.kind_of?(Net::HTTPSuccess)
+      @success ||= http_response.is_a?(Net::HTTPSuccess)
```

Comment:
Are you sure it will always be `Net::HTTPSuccess` if it is a child of it but a success response, this will be `false`. Is that the behaviour expected?